### PR TITLE
Css variables

### PR DESCRIPTION
**Pull request by iemeka**

### CODE REVIEW

**Review by dokasto — CHANGES_REQUESTED** [review]
Event delegation should only be used when you want to listen for an event in a lot of elements that are the same.
Right now we are using the same event handler for both input `type="color"` and `type="range"`

I'd say bind different event listeners to both types.

File: css_variable/index-START.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  <script>
+    const input = document.querySelector('.controls');
+
+    function handleUpdate(e){
+      if (e.target.tagName !== "INPUT") return;
+      const suffix = e.target.dataset.sizing || ""
+      document.documentElement.style.setProperty(`--${e.target.name}`,e.target.value + suffix)
+    }
+
+    input.addEventListener('change', handleUpdate);
```
Comment (dokasto) [inline css_variable/index-START.html:70]: Binding a single event handler on multiple elements of different type is an anti-pattern.

Comment (iemeka) [reply css_variable/index-START.html:70]: okay

**Comment by iemeka** [comment]
made changes

**Pull request closed without merging**